discord/resource_discord_channel.go

```diff
@@ -15,20 +15,24 @@ import (
 func getChannelSchema(channelType string, s map[string]*schema.Schema) map[string]*schema.Schema {
 	addedSchema := map[string]*schema.Schema{
 		"server_id": {
-			Type:     schema.TypeString,
-			Required: true,
+			Type:        schema.TypeString,
+			Required:    true,
+			Description: "ID of server this channel is in.",
 		},
-		"channel_id": {
-			Type:     schema.TypeString,
-			Computed: true,
+		"id": {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: "The ID of the channel.",
 		},
-		"category": {
-			Type:     schema.TypeString,
-			Optional: true,
+		"channel_id": {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: "The ID of the channel.",
 		},
```

Comment:
Why did you remove `category`?

Reply:
A few lines below, `category` is added if the channel type is not category, so this particular category definition is only used in category channels https://github.com/Lucky3028/terraform-provider-discord/blob/10f2774e5cf052f7b6c390ccc5b914c66c92b61a/discord/resource_discord_channel.go#L69-L74

In hindsight, this is technically a breaking change from a schema perspective and so I'm fine with reverting this if desired, but there is a check that would cause it to error if someone had provided this option on a category channel before https://github.com/Lucky3028/terraform-provider-discord/blob/10f2774e5cf052f7b6c390ccc5b914c66c92b61a/discord/resource_discord_channel.go#L94-L98

Reply:
Alright, we are on the same page. You don’t have to revert.
I couldn't find any changes similar to this one, are there others?

Reply:
Nope, I believe this is the only structural schema change

Reply:
Thanks!